### DISCUSSION TITLE
What is your favorite Manga?

### DISCUSSION THREAD

**Discussion by kkkyk373** [NAIST-SE/AttractingContributors#941] [General]
There are many Manga in Japan.
Please tell me the title of your favorite Manga.
My repository is here: 
https://github.com/ramu13/AttractingContributors-Manga

**Comment by bconfessor** [comment]
Done! Could you tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md

**Comment by k-mizoguch1** [comment]
plz
https://github.com/kzu000/Food-what-do-you-like
